.github/workflows/build.yml

```diff
@@ -664,11 +664,6 @@ jobs:
           path: ./TestResults/*.trx
           if-no-files-found: error
 
-      - name: Upload Code Coverage
```

Comment:
Code cov seems to be already failing on `main`:

```
info - 2025-02-08 03:42:07,424 -- ci service found: github-actions
warning - 2025-02-08 03:42:07,502 -- xcrun is not installed or can't be found.
warning - 2025-02-08 03:42:08,049 -- No gcov data found.
warning - 2025-02-08 03:42:08,065 -- coverage.py is not installed or can't be found.
info - 2025-02-08 03:42:08,862 -- Found 2 coverage files to report
info - 2025-02-08 03:42:08,862 -- > D:\a\bicep\bicep\codecov.exe.SHA256SUM
info - 2025-02-08 03:42:08,862 -- > D:\a\bicep\bicep\codecov.exe.SHA256SUM.sig
info - 2025-02-08 03:42:09,222 -- Process Upload complete
error - 2025-02-08 03:42:09,222 -- Upload failed: {"message":"Token required because branch is protected"}
```

So I went ahead and dropped it.